### PR TITLE
feat: remove payment

### PR DESCRIPTION
**Pull request by sofiavalenzuelapugh**

### CODE REVIEW

File: demo/loader.js [hunk 1]
```diff
@@ -163,7 +163,7 @@ document.addEventListener("DOMContentLoaded", async () => {
     // This helper calls our demo marketplace backend to retrieve the authToken.
     // You can manually set an auth token here for development purposes.
     apiKey: "api-key-123",
-    externalVendorId: "vendor-id-123",
+    externalVendorId: "wendys",
```
Comment (PabloReszczynski) [inline demo/loader.js:166]: Was this change on purpose?

**Review by PabloReszczynski — APPROVED** [review]

File: demo/loader.js (hunk 1 above)
Comment (sofiavalenzuelapugh) [reply demo/loader.js:166]: yes, is easier for testing with the demo
